e2e/conf/local.conf.js

```diff
@@ -16,15 +16,15 @@ const specs = [
 	// './experiences/facebook/facebook.test.js'
 	// './experiences/paywall/paywall.test.js'
 	// './experiences/featured_article/featured-article.test.js'
-	// './experiences/homepage/homepage.test.js'
+	'./experiences/homepage/homepage.test.js'
 	// './experiences/multicard/multicard.test.js'
 	// './experiences/newsletter/newsletter.test.js'
 	// './experiences/podcast/podcast.test.js'
 	// './experiences/push_notifications/push-notifications.test.js'
 	// './experiences/taboola/taboola.test.js'
 	// './experiences/top_articles/top-articles.test.js'
 	// './e2e/experiences.test.js'
-	'./e2e/relevant-experiences.test.js'
+	// './e2e/relevant-experiences.test.js'
```

Comment:
I guess you need to revert this

Reply:
yes i can revert if we need this local config to keep any particular shape

Reply:
> yes i can revert if we need this local config to keep any particular shape
Oh, is the local one. The local one no problem ;)